common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksIndex.scala

```diff
@@ -95,7 +95,7 @@ object WorksIndex {
   def genre(fieldName: String) = objectField(fieldName).fields(
     textField("label"),
     keywordField("ontologyType"),
-    identified("concepts", concept)
+    identified("concepts", rootConcept)
```

Comment:
So `genre.concept` can now be a `concept`, `agent` or `period`?

Not quite sure from the description what issue this is addressing, could you just clarify for me?

Reply:
It is a concept, of which agent and period inherit their properties.
So the mapping needs to allow for fields from all of those types.

So this is basically saying allow genre to have any of the fields from concept, agent or period.

Not very clear, I will refactor once this is in and we have the data working.